### ISSUE TITLE
Celery beat doesn't update scheduled tasks at runtime on changing of tasks through the django_celery_beat

### ISSUE THREAD

**Issue by AlexanderKaluzhny** [AlexanderKaluzhny/instanotifier#1]
Celery.beat.Scheduler doesn't update the self.schedule in the tick() method.

**Comment by AlexanderKaluzhny** [comment]
Is related to [django_celery_beat issue #7](https://github.com/celery/django-celery-beat/issues/7)
Proposed PR is [Populate heap when periodic tasks are changed #3791](https://github.com/celery/celery/pull/3791)

**Comment by AJRenold** [comment]
Hi! This sounds like the issue that we ran into. The fix that I created was just merged into django-celery-beat master - https://github.com/celery/django-celery-beat/pull/25

**Comment by AlexanderKaluzhny** [comment]
Waiting on an update of django-celery-beat.

**Comment by AlexanderKaluzhny** [comment]
django-celery-beat dependency removed. Closing the issue.
